studio/src/mas-side-nav.js

```diff
@@ -77,19 +81,17 @@ class MasSideNav extends LitElement {
                     label="Home"
                     value="home"
                     @click="${navigateToPage('welcome')}"
-                    selected
+                    ?selected=${this.currentPage.value === 'welcome'}
                 >
                     <sp-icon-home slot="icon"></sp-icon-home>
                 </sp-sidenav-item>
-
-                <sp-sidenav-item label="Promotions" value="promotions">
-                    <sp-icon-promote slot="icon"></sp-icon-promote>
-                </sp-sidenav-item>
-
-                <sp-sidenav-item label="Reporting" value="reporting">
-                    <sp-icon-graph-bar-vertical
-                        slot="icon"
-                    ></sp-icon-graph-bar-vertical>
+                <sp-sidenav-item
+                    label="Placeholders"
+                    value="placeholders"
+                    @click="${navigateToPage('placeholders')}"
```

Comment:
time to make these values (placeholders, content, welcome) a constant?

Reply:
done